go-controller/pkg/ovn/gateway_init.go

```diff
@@ -14,13 +14,19 @@ import (
 )
 
 // gatewayInit creates a gateway router for the local chassis.
-func gatewayInit(nodeName string, clusterIPSubnet []*net.IPNet, hostSubnets []*net.IPNet, joinSubnets []*net.IPNet, l3GatewayConfig *util.L3GatewayConfig, sctpSupport bool) error {
+func gatewayInit(nodeName string, clusterIPSubnet []*net.IPNet, hostSubnets []*net.IPNet, joinSubnets []*net.IPNet,
+	l3GatewayConfig, hybridGatewayConfig *util.L3GatewayConfig, sctpSupport bool) error {
 	// Create a gateway router.
 	gatewayRouter := gwRouterPrefix + nodeName
 	physicalIPs := make([]string, len(l3GatewayConfig.IPAddresses))
 	for i, ip := range l3GatewayConfig.IPAddresses {
 		physicalIPs[i] = ip.IP.String()
 	}
+	if config.Gateway.Mode == config.GatewayModeHybrid {
```

Comment:
Shouldn't this be `if hybridGatewayConfig != nil`?

Reply:
done